### PR TITLE
Added recipe for mongoengine

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

**Comment by pmlandwehr** [comment]
@jakirkham looks like this build (& others) are failing because ebooklib has gotten folded into the Appveyor build process. You've mentioned rebasing as a fix for this, and I'm not sure about pulling that off. I *have* merged in all upstream commits from conda-forge/staged-recipes to no end. What would your rebase prescription be?

**Comment by jakirkham** [comment]
So I'm working on a way for these merged recipes to be ignored in the CIs with PR ( https://github.com/conda-forge/staged-recipes/pull/1373 ). I'm hoping I can get that together soon so that this oddity won't affect anyone any more.

**Comment by jakirkham** [comment]
As far as a current solution, have you fetched `upstream` or similar recently? Maybe it is just out-of-date locally. 😕

File: recipes/mongoengine/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pymongo >=2.7.1
```
Comment (jakirkham) [inline recipes/mongoengine/meta.yaml]: Could we drop `pymongo` from the build dependencies?

**Comment by pmlandwehr** [comment]
(ping @jakirkham for merge)

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Pull request merged**